Frontend/src/Modal.svelte

```diff
@@ -16,7 +16,7 @@
 	<div on:click|stopPropagation>
 
 		<slot name="header" />
-		<hr />
+			<hr />
```

Comment:
OPINION: Tags like hr and br should be avoided if possible because the hold little to no semantic meaning on their own and only act as a target for styles. 99% of the time, you should be able to achieve the same effect by targeting elements nearby to cause a linebreak, spacing, horizontal lines, etc.. I'm not really against the hr tag specifically tbh and this is definitely a good use of it, I just want to let you know

Reply:
I definitely agree with <br>s. However I think <hr> is very useful and is just good at doing it's job. I have personally never seen a horizontal row, those simple lines, be anything but. I am curious about alternatives if you could lead me to some.

Reply:
Writing this on the fly and this might not fit your specific use-case, but essentially you could do something like this:

.modal-buttons{
   //current styles
    margin-top: 16px; //or whatever it currently is + 16px  
    border-top: 3px solid #777;
}

Reply:
But to be clear, I also think your use of \<hr\> is appropriate. I personally avoid pre-styled style-only tags like this, no matter the tag, because I think html shouldn't dictate styles, but that doesn't mean you have to too. You're using it as it was intended to be used.